src/entities/crm/crm.ts

```diff
@@ -337,6 +341,32 @@ export class CRM extends UniverseEntity<CRMPayload, CRMRawPayload> {
       throw this.handleError(new CRMAssociateUsersError(undefined, { error: err }))
     }
   }
+
+  public async createExternalUserFromPerson (payload: PersonPayload): Promise<number | undefined> {
```

Comment:
nit: see above: why not just accept a string value, called person?

Reply:
The promise response should not be a number, but the created channel user payload

Reply:
done